tests/auth/tokens.ts

```diff
@@ -1,20 +1,39 @@
+import * as jsonwebtoken from 'jsonwebtoken';
+
+const johnDoe = {
+	username: 'johndoe1',
+	email: 'johndoe@johndoe.com',
+	gitlab_id: 1325,
+	social_service_account: null,
+	hasPasswordSet: true,
+	needsPasswordReset: false,
+	public_key: false,
+	features: [],
+	id: 1344,
+	intercomUserHash:
+		'e03778dd29e157445f272acc921170cf2810b62f502645265cc349d6deda3524',
+	permissions: [],
+};
+const janeDoe = {
+	id: 152,
+	username: 'janedoe',
+	email: 'janedoe@asdf.com',
+	social_service_account: null,
+	has_disabled_newsletter: true,
+	hasPasswordSet: true,
+	needsPasswordReset: false,
+	public_key: false,
+	features: [],
+	intercomUserHash:
+		'0b4f9eb44b371f0e328ef5fe03ad7eb2f5f72dd418ef23149d5287096558ce03',
+	permissions: [],
+};
+
 export default {
 	johndoe: {
-		token:
-			'eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJ1c2VybmFtZSI6ImpvaG5kb2UxIiwiZW1haWwiOiJqb2huZG9lQGpvaG5kb2UuY29tIiwiZ2l0bGFiX2lkIjoxMzI1LCJzb2NpYWxfc2VydmljZV9hY2NvdW50IjpudWxsLCJoYXNQYXNzd29yZFNldCI6dHJ1ZSwibmVlZHNQYXNzd29yZFJlc2V0IjpmYWxzZSwicHVibGljX2tleSI6ZmFsc2UsImZlYXR1cmVzIjpbXSwiaWQiOjEzNDQsImludGVyY29tVXNlckhhc2giOiJlMDM3NzhkZDI5ZTE1NzQ0NWYyNzJhY2M5MjExNzBjZjI4MTBiNjJmNTAyNjQ1MjY1Y2MzNDlkNmRlZGEzNTI0IiwicGVybWlzc2lvbnMiOltdLCJpYXQiOjE0MjY3ODMzMTJ9.v5bmh9HwyUZu8zhh1rA79mTL-1jzDOO8eUr_lVaBwhg',
-		data: {
-			email: 'johndoe@johndoe.com',
-			username: 'johndoe1',
-			id: 1344,
-		},
+		token: jsonwebtoken.sign(johnDoe, 'very-secret'),
```

Comment:
Due to a bug fix in balena-request the token refresh now works properly in cases it was previously broken and would not be called, to match the previous behavior for testing I've switched these tokens to be generated on the fly so they are new and do not trigger a refresh (previous tokens had an `iat` some time in 2015)